README.md

```diff
@@ -67,8 +69,7 @@ To get started, see the following example:
 1. [Cats vs. Dogs](examples/Cats_v_Dogs_Test_Example.ipynb): Dataset from combined directory + CSV
 
 Examples coming soon:
-1. [Facebook Like Prediction](examples/Facebook_Like_Predictor.ipynb): Dataset from unsupervised directory only, with PCA visualization
```

Comment:
We've decided to not do these? :)

Reply:
Correct! I may do another example, but I felt like saying "Example coming soon" after a year is a little false advertising :D